### PR TITLE
fix(banner): display only for some subsidiaries

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Display only for some subsidiaries

MANAGER-1364

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#4530.0 ✘  
* init
  * deps ✘

**Comment by ovh-cds** [comment]
CDS Report build-manager#12.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✔ 
  * Build manager EU ✔ 
  * Build manager US ✔ 
  * Tests ✘

**Comment by ovh-cds** [comment]
CDS Report build#4536.0 ✘  
* init
  * deps ✘

File: client/app/configuration/configuration.controller.js [hunk 1]
```diff
@@ -37,6 +37,14 @@ angular.module('App').controller(
 
     buildSummitData() {
       this.localeForSummitBanner = this.$translate.use().split('_')[0] === 'fr' ? 'fr' : 'en';
+
+      const subsidiariesWithSummitBanner = ['FR', 'GB', 'DE', 'ES'];
+      this.shouldDisplayBanner = false;
+      return this.User
```
Comment (antleblanc) [inline client/app/configuration/configuration.controller.js]: So buildSummitData` method returns now a `Promise`. You can easilly do a `$q.all()` or chain both promises you have in `$onInit` method, Right?

But pay attention L19, you already fetch the current user;)

Comment (FredericEspiau) [reply client/app/configuration/configuration.controller.js]: I know all of this, it's supposed to be temporary and quickly done

**Pull request merged**